### PR TITLE
feat(account): execute multicall transactions

### PR DESCRIPTION
**Pull request by delaaxe**
- supports executing multiple calls in `Account#execute`
- sends those as a single StarkNet transaction to the account contract's `__execute__` method
- updates existing tests to support the new account
- adds a new case to test an actual multicall scenario
- improves error reporting

### CODE REVIEW

**Comment by dhruvkelawala** [comment]
Really happy about this PR! Was waiting for multicall execute for a while now. Thanks @delaaxe .

File: src/utils/transaction.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { ParsedStruct } from '../contract';
+import { Call } from '../types';
+import { getSelectorFromName } from './hash';
+import { BigNumberish, bigNumberishArrayToDecimalStringArray, toBN } from './number';
+
+export const fromCallsToCallArray = (calls: Call[]) => {
```
Comment (0xs34n) [inline src/utils/transaction.ts:6]: I would suggest adding some JSDoc to the function what it does and why it's there. And maybe renaming the function.

`transformCallsToMultiCallsArray()` might be more descriptive

**Comment by delaaxe** [comment]
@0xs34n resolved your suggestions and made a few more changes!

**Comment by dhruvkelawala** [comment]
Looks good to me. Waiting patiently for this to be merged and released :)

**Review by MilGard91 — APPROVED** [review]
Looks good to me

**Pull request merged**